common/src/test/java/com/revenuecat/purchases/common/DeviceCacheTest.kt

```diff
@@ -709,6 +628,38 @@ class DeviceCacheTest {
 
     // endregion
 
+    // region offerings response
+
+    @Test
+    fun `gets offerings from shared preferences`() {
+        every { mockPrefs.getString(offeringsResponseCacheKey, null) } returns "{\"test-key\": \"test-value\"}"
+        val offeringsResponse = cache.getOfferingsResponseCache()
+        assertThat(offeringsResponse).isNotNull
+        assertThat(offeringsResponse!!.getString("test-key")).isEqualTo("test-value")
```

Comment:
Do the tests continue executing after a failed assertion? This might be better
```suggestion
        assertThat(offeringsResponse?.getString("test-key")).isEqualTo("test-value")
```
If that's `null` that assertion would fail too anyway.

Reply:
Yeah, if an assertion fails, it doesn't continue, so that makes sense 👍